nodejs-bundle-babel-preset-es2015/babel-preset-es2015-6.6.0-registry.npmjs.org.tgz

```diff
@@ -0,0 +1 @@
+../.git/annex/objects/97/70/SHA256E-s314319--50075c5fa4399737aff808d3297e184a59e1dfd3422c05c4286ef6aa4d0e2f6d.org.tgz/SHA256E-s314319--50075c5fa4399737aff808d3297e184a59e1dfd3422c05c4286ef6aa4d0e2f6d.org.tgz
```

Comment:
I think we can live with that for now, given all of the other sources are coming from the NPM site, it's a good reduction. I'll have a think to see if it can be improved in future.